code/game/objects/items/robot/items/tools.dm

```diff
@@ -182,22 +182,31 @@
 	desc = "Equipment for cyborgs. Lifts objects and loads them into cargo."
 	icon = 'icons/psychonaut/mob/silicon/robot_items.dmi'
 	icon_state = "clamp"
+	/// The clamps owner.
 	var/mob/living/silicon/robot/host
-	var/list/stored_crates = list()
+	/// A list of the that clamp can carry.
+	var/list/can_carry = list(/obj/structure/closet/crate, /obj/item/delivery/big)
+	/// A list of the that clamp now carrying.
+	var/list/stored_things = list()
 	/// Time it takes to load a crate.
 	var/load_time = 3 SECONDS
```

Comment:
3 upgrade taktığını ve emaglı olduğunu düşünürsek 0.375 saniyede birini kaçırması hoş olmaz reaksiyon veremez bile
en azından `load_time`'ı 4 saniye yapıp upgradeyi 1.5 yapabilirsin 2 yerine dengeli olması için veya özellikle insanları kaçırırken yavaşlatma ekleyebilirsin ama yine de 0.375 crate için de az bi süre